### ISSUE TITLE
BartAttention中的attention_mask好像没做转换

### ISSUE THREAD

**Issue by Ningibu** [wzh9969/contrastive-htc#30]
BartAttetion 计算时，attention_mask未转换成-inf，相当于所有位置都加了1，实际transformer库中bert模型会将0，1的mask矩阵转换成torch中的min，代码中好像没看到这一步，实际跑的时候所有token_probs都是一致的

**Comment by wzh9969** [comment]
感谢您的指正！初步实验发现修正此错误后在WOS上的效果有略微提高，在`lamb=0.05,thre=0.02,seed=3`情况下达到macro-f1=81.48和micro-f1=87.17。但因可复现原因我们暂时不会更新代码，我们会尽快确定在其他数据集上的影响并更新说明。如果您想使用代码可以简单将所有attention_mask的输入改变为`None`，或是仿照BERT的处理添加一行`attention_mask = (1.0 - attention_mask) * -10000.0`。我们只对单token进行操作，attention_mask其实没有必要使用。
